### PR TITLE
Invoice api integration

### PR DESCRIPTION
**Pull request by reznord**

### CODE REVIEW

File: src/containers/widget/index.js [hunk 1]
```diff
@@ -1,35 +1,64 @@
...
+
+    const invoiceOptions = {
+      value,
+      apiToken: API_TOKEN,
+      baseURL: apiBaseUrl,
+      paymentRequestRenderer: paymentPagetRenderer,
+    };
+    const invoiceService = new InvoiceService(invoiceOptions);
+
+    invoiceService.requestPayment();
```
Comment (bumi) [inline src/containers/widget/index.js:59]: please not this currently returns a promise which resolves when the payment is received. - then we should display the success screen.

Comment (bumi) [reply src/containers/widget/index.js:59]: line 19/20 in the invoice service: https://github.com/bumi/area402-widget/pull/29/files#diff-11dec6989c54070d1f26940347dee3199af4b02aad997a4724b724c392dab984R19

Comment (reznord) [reply src/containers/widget/index.js:59]: yeah, I am aware of that I know the code, there's no callback that comes to this line here, it more like the whole functionality is passed down to the `paymentRequestRenderer` callback function.

Comment (reznord) [reply src/containers/widget/index.js:59]: > line 19/20 in the invoice service: https://github.com/bumi/area402-widget/pull/29/files#diff-11dec6989c54070d1f26940347dee3199af4b02aad997a4724b724c392dab984R19

Yes, I am aware of how the Invoice is implemented. Understood after going through the code.

Comment (bumi) [reply src/containers/widget/index.js:59]: we can change the implementation of the promise or what `requestPayment` does if needed and if that works better with the preact flow.

**Review by bumi — APPROVED** [review]

**Pull request merged**